src/presentation/react/core/components/toggle/Toggle.tsx

```diff
@@ -3,7 +3,7 @@ import _ from "lodash";
 
 interface InputParameters {
     disabled?: boolean;
-    label: string;
+    label: string | React.ReactNode;
```

Comment:
A string is also a React.ReactNode (AFIK)

Reply:
Didn't know about it :o well, it make sense at the end of the day